docs/docs-content/clusters/edge/edgeforge-workflow/palette-canvos/palette-canvos.md

```diff
@@ -223,6 +223,15 @@ customization.
    cat user-data
    ```
 
+   :::info
```

Comment:
@lennessyy We already have a healthy number of admonitions on this page. I do see the value in the admonition, so maybe we make it a tip vs an infobox? We already have quite a few info boxes, and it kinda feels like a tip.